README.md

```diff
@@ -66,10 +66,9 @@ reverse proxy, all running on the same system (requires
 [puppetlabs/postgresql](https://forge.puppetlabs.com/puppetlabs/postgresql) and
 [jfryman/nginx](https://forge.puppetlabs.com/jfryman/nginx)):
 
-````puppet
+```puppet
 class { 'postgresql::server':
-  listen_addresses        => '127.0.0.1',
```

Comment:
In some situations, setting this can cause Postgres to fail on a config syntax error. Since `127.0.0.1` is the default value, this can be omitted.